### PR TITLE
Delocalize files / directories in cwl.output.json

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

File: backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala [hunk 1]
```diff
@@ -143,11 +143,21 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
   def mapCommandLineWomFile(womFile: WomFile): WomFile =
     womFile.mapFile(workflowPaths.buildPath(_).pathAsString)
 
+  // This is a trickery to allow the mapCommandLineWomFile above to call isAdHoc file without creating an
```
Comment (mcovarr) [inline backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala]: a trickery => trickery

**Comment by Horneth** [comment]
@kshakir 
> Does this still mean that all other backend implementations should still look for cwl.output.json as a glob and not a regular file?

I think `cwl.output.json` should be looked at as an "optional output". At least for delocalization purposes (since the actual file is never used as is.. I think..). With PAPI V2 we should be able to support optional outputs and so when that happens we should switch from using a glob based approach to an optional approach. For backends that can't support optional outputs for whatever reason, the glob way is still viable IMO.
This is not to say that we don't need a larger glob refactoring, but we've been using glob as a workaround for optional outputs in some cases because of restrictions of V1 that I think we should not impose on V2, or any backend that can deal cleanly with optional outputs.

> As the conformance tests are being removed, does that infer that as of this PR is CWL not officially supported on PapiV1?
yes I think there's no official plan to support CWL on V1 (@geoffjentry is that true ?) Since this particular test can only work on V2, instead of duplicating the `papi_conformance_expected_failures.txt` file with a V1/V2 I took the opportunity to nix V1 from travis.

File: backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala [hunk 2]
```diff
@@ -143,11 +143,21 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
...
+  // infinite recursion. This should really go away when we finally can have a sane implementation
+  // keeping track of the paths cleanly without so many value mappers
```
[the remark this answers is not in the document]
Comment (mcovarr) [reply backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala]: I second that, anything to expedite the mergification of this PR with its PAPI 1 conformance elimination. I need that for my optional outputs stuff. 🙂

[the remark this answers is not in the document]
Comment (Horneth) [reply backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala]: Merged - I'll follow up with a PR to address this and your comment @mcovarr

**Pull request merged**